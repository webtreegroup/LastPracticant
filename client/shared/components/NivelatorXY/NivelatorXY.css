.nivelator-xy {
    height: 100%;
}
