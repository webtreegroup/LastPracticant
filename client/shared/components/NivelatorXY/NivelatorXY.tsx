import { Grid } from '@material-ui/core';
import bem from 'bem-cn';
import { ComponentCommonProps } from 'client/shared/types';
import React, { FC, memo } from 'react';
import './NivelatorXY.css';

const block = bem('nivelator-xy');

export const NivelatorXY: FC<ComponentCommonProps> = memo(
    ({
        children,
        className,
    }) => (
        <Grid
            container
            className={block({}).mix(className).toString()}
            justify="center"
            alignItems="center"
            direction="column"
        >
            {children}
        </Grid>
    ),
);
