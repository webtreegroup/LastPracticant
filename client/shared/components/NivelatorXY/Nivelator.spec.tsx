import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { NivelatorXY } from './NivelatorXY';

afterEach(cleanup);

describe('NivelatorXY', () => {
    it('Render NivelatorXY component', () => {
        const { asFragment } = render(<NivelatorXY />);
        expect(asFragment()).toMatchSnapshot();
    });
});
