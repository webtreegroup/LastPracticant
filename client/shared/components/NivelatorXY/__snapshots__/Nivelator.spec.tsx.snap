// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NivelatorXY Render NivelatorXY component 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root nivelator-xy MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  />
</DocumentFragment>
`;
